sale_stock_available_to_promise_release/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Camptocamp (https://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    sale_date_expected = fields.Date(
+        string="Delivery Date", related="group_id.date_expected", store="True"
```

Comment:
`store=True`?

Reply:
Fixed, rebased and force-pushed with this change